kube/services/jobs/usersync-cronjob.yaml

```diff
@@ -181,7 +213,13 @@ spec:
                   exitcode=$?
                 else
                   if [[ -f /mnt/shared/user.yaml ]]; then
-                    output=$(fence-create sync --arborist http://arborist-service --sync_from_dbgap "$SYNC_FROM_DBGAP" --projects /var/www/fence/projects.yaml --yaml /mnt/shared/user.yaml 2>&1)
+                    if [ ! -z "$DBGAP_BACKUP_BUCKET" || ! -z "$DBGAP_DOWNLOAD_FOLDER" ]; then
```

Comment:
i think you need `&&` not `||`?
here and in all the other places you check these

Reply:
i just realized `DBGAP_DOWNLOAD_FOLDER` is optional while reading the fence code. so i think this should be:
`if [ ! -z "$DBGAP_BACKUP_BUCKET" ]; then`
because if the folder is provided but not the bucket, it won't work